src/components/Molecules/NavbarLeft/NavbarLeft.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render, cleanup } from '@testing-library/react';
+import NavbarLeft from './index';
+
+describe('Test NavbarLeft Component', () => {
+  afterEach(cleanup);
+  it('renders without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<NavbarLeft />, div);
```

Comment:
No need to use `ReactDOM.render` in here, in fact you could delete this `it` case and [write fewer, but longer tests](https://kentcdodds.com/blog/write-fewer-longer-tests).

Reply:
Got it! Removed all those tests with regards to component rendering and merged some others in just a single test case.